_data/simple-icons.json

```diff
@@ -7016,6 +7016,11 @@
             "hex": "0097C2",
             "source": "https://opennebula.io/docs/"
         },
+        {
+            "title": "OpenSea",
+            "hex": "2081E2",
+            "source": "https://docs.opensea.io/docs/logos"
```

Comment:
can you change the source to `https://docs.opensea.io` only please.

Reply:
I've changed the url in latest commit.

Reply:
I think it makes sense to link to https://docs.opensea.io/docs/logos. That seems like a fairly permanent link.